src/impl_/pymodule.rs

```diff
@@ -71,6 +71,21 @@ impl ModuleDef {
         panic_result_into_callback_output(
             py,
             std::panic::catch_unwind(move || -> PyResult<_> {
+                #[cfg(all(PyPy, not(Py_3_8)))]
+                {
+                    const PYPY_GOOD_VERSION: [u8; 3] = [7, 3, 9];
```

Comment:
Maybe I don't understand?

```suggestion
                    const PYPY_GOOD_VERSION: [u8; 3] = [7, 3, 8];
```

Reply:
Haha good catch, I set it higher when manually testing 😅

Reply:
:+1: